searchcore/src/vespa/searchcore/proton/matching/i_ranking_assets_repo.h

```diff
@@ -4,15 +4,18 @@
 
 #include <vespa/eval/eval/value_cache/constant_value.h>
 
+namespace search::fef { class OnnxModel; }
 namespace proton::matching {
 
 /**
  * Interface for retrieving a named constant rank value to be used by features in the rank framework.
  * If the given value is not found a nullptr should be returned.
  */
-struct IConstantValueRepo {
+struct IRankingAssetsRepo {
```

Comment:
Please update the interface description.

Reply:
Fixed